src/app/accounts/sso.component.html

```diff
@@ -0,0 +1,9 @@
+<form id="sso-page" (ngSubmit)="submit()">
+    <div class="content">
+        <img class="logo-image" alt="Bitwarden">
+        <div class="box">
+            <i class="fa fa-spinner fa-spin" title="{{'loading' | i18n}}" aria-hidden="true"></i>
+            {{'loggingInPleaseWait' | i18n}}
```

Comment:
Just re-use existing "loading" string. That's what I ended up doing in bwdc.

Reply:
Fixed